src/clj_jargon/init.clj

```diff
@@ -48,18 +49,37 @@
   (log/debug curr-with-jargon-index "- returning a proxy input stream...")
   (proxy-input-stream cm retval))
 
+(defn override-user-account
+  [cfg user pass]
+  (IRODSAccount. (:host cfg)
+    (Integer/parseInt (:port cfg))
+    user
+    pass
+    (:home cfg)
+    (:zone cfg)
+    (:defaultResource cfg)))
```

Comment:
This is just moved from below to keep all the account stuff together, basically.